orm/internal/testpb/test_schema.proto

```diff
@@ -7,31 +7,33 @@ import "google/protobuf/duration.proto";
 import "cosmos/orm/v1/orm.proto";
 
 message ExampleTable {
+  // clang-format off
```

Comment:
not a crazy fan about these. it seems to add overhead in the .pb.go files

Reply:
It does not add anything in the `.pb.go`. `make proto-all` added unrelated changes. If you like, I can only run `make proto-format` in this issue.

Reply:
ah nvm ignore the comment. Was weird how the bytes changed